prov/efa/src/efa_mr.c

```diff
@@ -212,6 +213,12 @@ static int efa_mr_hmem_setup(struct efa_mr *efa_mr,
 							 uint64_t flags)
 {
 	int err;
+	struct iovec mr_iov = {0};
+
+	if (flags & FI_MR_DMABUF)
+		ofi_mr_get_iov_from_dmabuf(&mr_iov, attr->dmabuf, 1);
+	else
+		memcpy(&mr_iov, attr->mr_iov, sizeof(*attr->mr_iov));
```

Comment:
Would it be faster to assign the fields of `mr_iov` instead of memcpy?

Reply:
Updated